.github/workflows/run-integration-test-dispatch.yaml

```
@@ -0,0 +1,87 @@
+# Workflow Dispatch to run terra-ui integration tests
+# If trying to run terra-ui tests as part of CI from other GHA, use the run-integration-tests workflow_call instead.
+name: External terraui Test
```

Super nit: the all lower case "terraui" doesn't match the other two capitalized words.

```suggestion
name: External Terra UI Test
```